src/lib/wapi/functions/send-message.js

```diff
@@ -1,14 +1,12 @@
 export function sendMessage(id, message, done) {
-  var chat = WAPI.getChat(id);
-  if (chat) {
+  let chat = WAPI.getChat(id);
```

Comment:
Is this actually working when the chat is not existing? I personally use WAPI `sendMessageToId` that fetches the contact from the contact list and sends a message to him

Reply:
Exposed method to sulla now